api/Headers.json

```diff
@@ -0,0 +1,1106 @@
+{
+  "api": {
+    "Headers": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Headers",
+        "support": {
+          "webview_android": {
+            "version_added": "42"
```

Comment:
I think this is the complete thing for the Chrome family.

```json
"webview_android": [
              {
                "version_added": "42"
              },
              {
                "version_added": "41",
                "flags": [
                  {
                    "type": "preference",
                    "name": "Experimental Web Platform Features"
                  }
                ]
              }
            ],
            "chrome": [
              {
                "version_added": "42"
              },
              {
                "version_added": "41",
                "flags": [
                  {
                    "type": "preference",
                    "name": "Experimental Web Platform Features"
                  }
                ]
              }
            ],
            "chrome_android": [
              {
                "version_added": "42"
              },
              {
                "version_added": "41",
                "flags": [
                  {
                    "type": "preference",
                    "name": "Experimental Web Platform Features"
                  }
                ]
              }
            ],
```

Reply:
👍